src/lib/chains/algorand/algoError.ts

```diff
@@ -0,0 +1,20 @@
+export enum AlgoError {
+    CLIENT_NOT_SET = 'CLIENT HAS NOT SET',
+    BRIDGE_NOT_SET = 'BRIDGE HAS NOT SET',
```

Comment:
I like the centralization of error, but please do not use all caps in string fields. Makes it seem like we are yelling at the developers. All caps is ok for variable constant names, not in strings,

Reply:
@andropixels do not add whitespace within variable value, use `_`